gh-128307:  support eager_start kwarg in create_eager_task_factory, and pass kwargs from asyncio.create_task and TaskGroup.create_task (#128306)
python/cpython: Doc/library/asyncio-eventloop.rst

   .. versionadded:: 3.5.2

> .. method:: loop.create_task(coro, *, name=None, context=None, eager_start=...)

Reviewer: Why not `eager_start=None` like the others?

Author: ```suggestion
.. method:: loop.create_task(coro, *, name=None, context=None, eager_start=None)
```

   .. versionadded:: 3.5.2

> .. method:: loop.create_task(coro, *, name=None, context=None, eager_start=None)

Author: ```suggestion
.. method:: loop.create_task(coro, *, name=None, context=None, eager_start=...)
```

   custom :class:`contextvars.Context` for the *coro* to run in.
   The current context copy is created when no *context* is provided.

   An optional keyword-only *eager_start* argument allows specifying
   if the task should execute eagerly during the call to create_task,

>    or be scheduled later. If the *eager_start* is not passed the mode set

Author: ```suggestion
   or be scheduled later. If *eager_start* is not passed the mode set
```

>    or be scheduled later. If the default ``None`` is passed the mode set
>    by :meth:`loop.set_task_factory` will be used.

Author: ```suggestion
   or be scheduled later. If the *eager_start* is not passed the mode set
   by :meth:`loop.set_task_factory` will be used.
```
